wapitiCore/definitions/exec.py

```diff
@@ -24,6 +24,8 @@
 NAME = _("Command execution")
 SHORT_NAME = NAME
 
+WSTG_CODE = ["WSTG-INPV-12"]
+
```

Comment:
Please update OWASP reference to https://owasp.org/www-project-web-security-testing-guide/latest/4-Web_Application_Security_Testing/07-Input_Validation_Testing/12-Testing_for_Command_Injection